### PR TITLE
Desktop: Fixed the Goto Anything item list overflow

### LINKED ISSUE #3095: Desktop: Goto Anything search results overflow
## Environment

Joplin version: 1.0.201
Platform: Windows

joplin gotoanything error

## Steps to reproduce

1. Press Ctrl+g or click on 'Goto Anything' under the Tools in menu.
2. Input something relevant 
3. If the no. of search results is large, the item list overflows.

## Describe what you expected to happen
The search results must not overflow.

### PR DESCRIPTION
**Pull request by Rahulm2310**
Fixes #3095 

**Before Changes:**

joplin gotoanything error

**After Changes:**

joplin gotoanything error solve

### CODE REVIEW

File: ElectronClient/plugins/GotoAnything.jsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ class Dialog extends React.PureComponent {
 		const itemHeight = this.state.resultsInBody ? 84 : 64;
 
 		this.styles_[styleKey] = {
-			dialogBox: Object.assign({}, theme.dialogBox, { minWidth: '50%', maxWidth: '50%' }),
+			dialogBox: Object.assign({}, theme.dialogBox, { minWidth: '50%', maxWidth: '50%', maxHeight: '80%', display: 'flex', flexDirection: 'column' }),
```
Comment (laurent22) [inline ElectronClient/plugins/GotoAnything.jsx:67]: Could you instead apply the fix to theme.dialogBox? (in theme.js) That way all dialog boxes will benefit from it. Also in that case, please check the other dialog boxes to make sure they still work.

Comment (Rahulm2310) [reply ElectronClient/plugins/GotoAnything.jsx:67]: Sure sir, I will do it.

**Comment by Rahulm2310** [comment]
@laurent22 I have made the required changes and all other dialog boxes are working perfectly fine.

**Comment by Rahulm2310** [comment]
@PackElend label me please

**Comment by laurent22** [comment]
That's great, thank you!

**Pull request merged**